### ISSUE TITLE
Can't access search in Obsidian [FIX]

### ISSUE THREAD

**Issue by edbock** [khoj-ai/khoj#623]
## Describe the bug
A clear and concise description of what the bug is. Please include what you were expecting to happen vs. what actually happened.
Thank you for the [Dataview fix](https://github.com/khoj-ai/khoj/commit/1a46734485ffe04c372c1857989d78c4081c6084). It's nice to have the chat window in Obsidian now. However, I no longer have access to the regular incremental search box.

Before: clicking the magnifying glass would bring up incremental search. Typing in the box would produce instant results.
After: clicking the magnifying glass brings up the chat window. I can't find the search window option. 

I tried creating a hotkey for search, and it does bring up a window but without the text entry portion. I can only select from my previous search results.

## Platform
- Server:
    - [ ] Cloud-Hosted (https://app.khoj.dev)
    - [ ] Self-Hosted Docker
    - [x] Self-Hosted Python package
    - [ ] Self-Hosted source code
- Client:
    - [x] Obsidian
    - [ ] Emacs
    - [ ] Desktop app
    - [ ] Web browser
    - [ ] WhatsApp
- OS:
    - [ ] Windows
    - [ ] macOS
    - [x] Linux
    - [ ] Android
    - [ ] iOS

### If self-hosted
- Server Version [e.g. 1.0.1]: 1.2.1

**Comment by debanjum** [comment]
1. I'd removed Khoj search and replaced it with Khoj chat in the side ribbon. Just kept one Khoj command on the side ribbon to avoid clutter and felt folks chat more than search. I can add it back if required
2. For now though you should be able to find `Khoj: Search` in your [Obsidian command palette](https://help.obsidian.md/Plugins/Command+palette). You can also bind Khoj: Search to a [Hotkey in Obsidian](https://help.obsidian.md/User+interface/Hotkeys) for easier access

> I tried creating a hotkey for search, and it does bring up a window but without the text entry portion. I can only select from my previous search results.

This sounds strange. Can you share more details, screenshots of what this modal looks like and the hotkey binding you did? There is a `Khoj: Find Similar Notes` command that allows you to find notes similar to the currently open note. The behavior you describe sounds somewhat similar to what you'd see when you run that command

**Comment by edbock** [comment]
Oops, my bad. I didn't look carefully enough at the hotkeys and indeed, that's what was happening. In fact, I'm very happy you pointed me back in that direction. The similar notes feature is fantastic, and is high on the list of features I would have requested.

If I hadn't been so focused on looking for and fixing issues I would have figured this out myself. That said, you might want to make these features a little more prominent in the docs.

I would say you are probably fine to leave search as the default. I agree having two buttons might clutter things up a bit too much. On the other hand, if it's not hard to do maybe you can add the option for users to select either/or, or both.

Thank you for your patience in helping me figure this out.

**Comment by debanjum** [comment]
Glad we resolved the confusion and you found the similar notes feature useful! It is mentioned in the [docs](https://docs.khoj.dev/clients/obsidian/#find-similar-notes) but we should add more details/images explaining the available commands for sure!
